pkg/executor/wasm/handler.go

```diff
@@ -64,13 +64,6 @@ type executionHandler struct {
 func (h *executionHandler) run(ctx context.Context) {
 	ActiveExecutions.Inc(ctx)
 	defer func() {
-		if r := recover(); r != nil {
-			h.logger.Error().
-				Str("recover", fmt.Sprintf("%v", r)).
-				Msg("execution recovered from panic")
-			// TODO don't do this.
-			h.result = &models.RunCommandResult{}
-		}
```

Comment:
If we are 100% sure the code called below here won't panic then I am for removing this defer statement, otherwise I'd find it safer to leave it here.

Reply:
If we leave it, then I'd suggest we change it to returning an error instead as at the moment it makes it look like the execution succeeded.

Reply:
That sounds good to me. Basically do what was done here:
```
	h.result = executor.NewFailedResult(
			fmt.Sprintf("unable to find the entrypoint '%s' in the WASM module", h.arguments.EntryPoint),
		)
		return
```
but change the error message to state a panic occurred?

Reply:
Done in 94b9c8e2dab677fd61d21b86fc8328ed6a488d20